code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -608,6 +608,46 @@
 	assignedrole = "Space Pirate"
 	var/rank = "Mate"
 
+/obj/effect/mob_spawn/human/pirate/on_attack_hand(mob/living/user, act_intent = user.a_intent, unarmed_attack_flags)
+	. = ..()
+	if(.)
+		return
+	if(user.mind.has_antag_datum(/datum/antagonist/pirate))
+		to_chat(user, "<span class='notice'>Your shipmate sails within their dreams for now. Perhaps they may wake up eventually.</span>")
+	else
+		to_chat(user, "<span class='notice'>If you want to kill the pirate off, something to pry open the sleeper might be the best way to do it.</span>")
+
+
+/obj/effect/mob_spawn/human/pirate/attackby(obj/item/W, mob/user, params)
+	if(W.tool_behaviour == TOOL_CROWBAR && user.a_intent != INTENT_HARM)
+		if(user.mind.has_antag_datum(/datum/antagonist/pirate))
+			to_chat(user,"<span class='warning'>Why would you want to do that to your shipmate? That'd kill them.</span>")
+			return
+		user.visible_message("<span class='warning'>[usr.name] pries open the [src], disrupting the sleep of the pirate within and killing them.</span>",
```

Comment:
[usr] can cause runtimes, since attackby doesn't always have a stacktrace pointing towards a defined usr. A much better way to handle this is by doing this
```suggestion
		user.visible_message("<span class='warning'>[user] pries open the [src], disrupting the sleep of the pirate within and killing them.</span>",
```

Reply:
Done with recent PR